src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/ControlChars.vb

```diff
@@ -0,0 +1,22 @@
+' Copyright (c) Microsoft. All rights reserved.
+' Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+Namespace Microsoft.VisualBasic
+    ' Contants for the Control Characters
+    Public NotInheritable Class ControlChars
+
+        Public Const CrLf As String = ChrW(13) & ChrW(10)
+        Public Const NewLine As String = ChrW(13) & ChrW(10)
```

Comment:
I do not know the answer at the moment. In any case this is out of scope for this commit.